app/views/users/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% @users.each do |user| %>
+    <p>
+      <%= user.email %><%= link_to "Add Friend", friendships_path(:friend_id => user), :method => :post %>
+    </p>
+  <% end %>
+</ul>
```

Comment:
Maybe add a if current_user (linked to the fact that you can see a list of emails and add friend buttons after you signed out)